LICENSE

```diff
@@ -1,7 +1,31 @@
-Node's license follows:
+Node.js is licensed for use as follows:
 
-====
+"""
```

Comment:
> Can we agree to merge this and update from master down when we have proper legal advice to work with?

Updating the LICENSE file when we know that we don't know what we're doing seems like a bad idea, especially if it's actually not needed to continue releasing v0.10.x versions. So instead I would suggest not merging these changes, and waiting until we have legal advice to update these files across all relevant branches.